homeassistant/components/myq/const.py

```diff
@@ -39,3 +43,36 @@
 # Estimated time it takes myq to complete a transition
 # from one state to another
 TRANSITION_COMPLETE_DURATION = 37
+
+MANUFACTURER = "The Chamberlain Group Inc."
+
+KNOWN_MODELS = {
```

Comment:
This seems like something that would benefit from being included in the PyPI lib.

Reply:
Added to the queue. Will send a PR upstream and switch this once its merged

Reply:
https://github.com/home-assistant/core/pull/35463
Done here